addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -166,6 +166,9 @@ Wysiwyg.include({
                     .write(`<!DOCTYPE html><html${
                         self.options.iframeHtmlClass ? ' class="' + self.options.iframeHtmlClass +'"' : ''
                     }>${iframeContent}</html>`);
+                // Closing the document might trigger a new 'load' event.
```

Comment:
There's a `avoidDoubleLoad` mechanism in place a few lines above, but it does not prevent the 'load' handler nor the `.then` callback to run twice, despite its name.

It would prevent the `.then` callback to run twice only if the `defAsset` promise was unresolved when the 'load' event handler was called, but I cannot see how that could happen, as the `defAsset` is `await`ed for on `willStart`, and `_loadIframe` (the function where this code lives) is only called by `startEdition` (which is called after the widget is mounted).


Anyway, the `avoidDoubleLoad` is read elsewhere and seems to be part of a fix of a very edgy case I don't know about, so I'm not removing it.